img/b/browserstack.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 29.7 31"><g fill="none"><path fill="#ecb360" d="M30.374 15.517c0 8.57-6.947 15.517-15.517 15.517S-.66 24.087-.66 15.517 6.287 0 14.857 0s15.517 6.947 15.517 15.517"/><path fill="#d76835" d="M26.236 13.966c0 7.712-6.021 13.965-13.448 13.965-7.427 0-13.448-6.253-13.448-13.965C-.66 6.253 5.36 0 12.788 0c7.427 0 13.448 6.253 13.448 13.966"/><path fill="#d33a41" d="M27.27 12.414c0 6.856-5.326 12.414-11.896 12.414-6.57 0-11.896-5.558-11.896-12.414S8.804 0 15.374 0c6.57 0 11.897 5.558 11.897 12.414"/><path fill="#b6cb46" d="M26.236 13.448c0 6.285-4.863 11.38-10.862 11.38S4.512 19.733 4.512 13.448c0-6.284 4.863-11.379 10.862-11.379s10.862 5.095 10.862 11.38"/><path fill="#66ad4a" d="M25.202 14.483c0 5.713-4.632 10.345-10.345 10.345-5.713 0-10.345-4.632-10.345-10.345 0-5.714 4.632-10.345 10.345-10.345 5.713 0 10.345 4.631 10.345 10.345"/><path fill="#aed7dc" d="M21.064 12.414a8.276 8.276 0 1 1-16.552 0 8.276 8.276 0 0 1 16.552 0"/><path fill="#5bb1cf" d="M22.098 11.38a7.241 7.241 0 1 1-14.483 0 7.241 7.241 0 0 1 14.483 0"/><path fill="#25a8c3" d="M22.098 11.896c0 3.714-2.779 6.725-6.207 6.725s-6.206-3.011-6.206-6.725c0-3.713 2.778-6.724 6.206-6.724 3.428 0 6.207 3.011 6.207 6.724"/><path d="M20.03 12.93c0 3.143-2.317 5.69-5.173 5.69-2.857 0-5.172-2.547-5.172-5.69 0-3.141 2.315-5.689 5.172-5.689 2.856 0 5.172 2.548 5.172 5.69"/><path fill="#1b1a18" d="M14.857 18.62c-2.857 0-5.172-2.547-5.172-5.69 0-3.141 2.315-5.689 5.172-5.689 2.856 0 5.172 2.548 5.172 5.69 0 3.142-2.316 5.69-5.172 5.69z"/><path fill="#fffffe" d="M17.68 11.29c-.493.824-1.46 1.3-2.162 1.063-.702-.236-.872-1.096-.38-1.92.492-.823 1.46-1.298 2.161-1.062.702.236.872 1.096.38 1.92"/></g></svg>
```

Comment:
Actually seems that GitHub's render of the image worked as intended.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 31.034 31.034"><path fill="#ecb360" d="M31.034 15.517c0 8.57-6.947 15.517-15.517 15.517S0 24.087 0 15.517 6.947 0 15.517 0s15.517 6.947 15.517 15.517"/><path fill="#d76835" d="M26.896 13.966c0 7.712-6.021 13.965-13.448 13.965C6.021 27.931 0 21.678 0 13.966 0 6.253 6.02 0 13.448 0c7.427 0 13.448 6.253 13.448 13.966"/><path fill="#d33a41" d="M27.93 12.414c0 6.856-5.326 12.414-11.896 12.414-6.57 0-11.896-5.558-11.896-12.414S9.464 0 16.034 0c6.57 0 11.897 5.558 11.897 12.414"/><path fill="#b6cb46" d="M26.896 13.448c0 6.285-4.863 11.38-10.862 11.38S5.172 19.733 5.172 13.448c0-6.284 4.863-11.379 10.862-11.379s10.862 5.095 10.862 11.38"/><path fill="#66ad4a" d="M25.862 14.483c0 5.713-4.632 10.345-10.345 10.345-5.713 0-10.345-4.632-10.345-10.345 0-5.714 4.632-10.345 10.345-10.345 5.713 0 10.345 4.631 10.345 10.345"/><path fill="#aed7dc" d="M21.724 12.414a8.276 8.276 0 1 1-16.552 0 8.276 8.276 0 0 1 16.552 0"/><path fill="#5bb1cf" d="M22.758 11.38a7.2415 7.2415 0 1 1-14.483 0 7.2415 7.2415 0 0 1 14.483 0"/><path fill="#25a8c3" d="M22.758 11.896c0 3.714-2.779 6.725-6.207 6.725s-6.206-3.011-6.206-6.725c0-3.713 2.778-6.724 6.206-6.724 3.428 0 6.207 3.011 6.207 6.724"/><path d="M20.69 12.93c0 3.143-2.317 5.69-5.173 5.69-2.857 0-5.172-2.547-5.172-5.69 0-3.141 2.315-5.689 5.172-5.689 2.856 0 5.172 2.548 5.172 5.69"/><path fill="#1b1a18" d="M15.517 18.62c-2.857 0-5.172-2.547-5.172-5.69 0-3.141 2.315-5.689 5.172-5.689 2.856 0 5.172 2.548 5.172 5.69 0 3.142-2.316 5.69-5.172 5.69z"/><path fill="#fffffe" d="M18.34 11.29c-.493.824-1.46 1.3-2.162 1.063-.702-.236-.872-1.096-.38-1.92.492-.823 1.46-1.298 2.161-1.062.702.236.872 1.096.38 1.92"/></svg>
```

Reply:
Thanks!